### DISCUSSION TITLE
Practicas de Intro

### DISCUSSION THREAD

**Discussion by ValencianoAlan** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#88] [Q&A] [answered]
Hola chavos, disculpen pero me podrían explicar bien que tenemos que hacer en la parte de requerimientos y en las demás practicas de la primer semana por favor, ya vi los lives pero no entendí mucho acerca de los requerimientos y del buyer

**Comment by doguedogue** [comment, marked as answer]
Mira tienes que ponerte en el rol que te piden hacer el sitio web abogabot con los requisitos que pusieron en la práctica.

Básicamente es como planear y acordar con tu cliente (el dueño del sitio de abogados) que le vas a entregar.

Tu buyer es la persona que va a usar el sitio web, alguien que quiere hacer una demanda.

Y así, usa las herramientas que nos enseñaron a usar y vas diseñando que vas a hacer para construir el sitio web.

**Reply by ValencianoAlan** [reply]
entonces respecto a la plantilla que nos pasaron, si en algún punto no se que poner, lo dejo así en blanco? y también otra cosa, los diagramas que están ahí mismo en que pagina se hacen

**Reply by buc-io** [reply]
Hola Alan para los diagramas puedes descargarte un software que se llama ClickCharts, ahí vienen diagramas BPMN que son parecidos al que tenemos en el ejemplo. 
Te dejo el recurso --> https://www.nchsoftware.com/chart/es/index.html 
(me parece que esta disponible tanto para Mac os y windows)

**Reply by ValencianoAlan** [reply]
va va muchas gracias a ambos, los tqm uwu
